test/new-e2e/utils/clients/aws.go

```diff
@@ -0,0 +1,61 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package clients
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	awsconfig "github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/ssm"
+)
+
+const (
+	awsTimeout = 5 * time.Second
+)
+
+var (
+	initLock     = sync.Mutex{}
+	awsConfig    *aws.Config
+	awsSSMClient *ssm.Client
+)
+
+// GetAWSSSMClient returns an aws SSM client
+func GetAWSSSMClient() (*ssm.Client, error) {
```

Comment:
Is this needed out of the aws credential manager ? If not, this code could become private and go into the aws credential manager file

Reply:
I believe it could be used to get some parameters in the CI. We'll know more when we have more tests and running in CI.